cpp/src/arrow/json/chunker.cc

```diff
@@ -46,53 +43,65 @@ namespace json {
 using internal::make_unique;
 using util::string_view;
 
-Status StraddlingTooLarge() {
+static string_view View(const std::shared_ptr<Buffer>& buffer) {
```

Comment:
`ViewFromBuffer`, for clarity?
(perhaps this could also be a conversion operator in the `Buffer` class...)

Reply:
I'll write the conversion operator